microservices-ftw.md

```diff
@@ -0,0 +1,51 @@
+Micro services For the Win========================
+
+* Speaker   : Nelson Gomes
+* Available : October 7th and 8th 
+* Length    : 30 to 60 minutes depending of available slots
+* Language  : English
```

Comment:
The language refers to the language the presentation is given and not the programming language.